### PR TITLE
add 0BSD License as allowed license

### PR DESCRIPTION
**Pull request by Uzlopak**
0BSD is a variation of ISC

So we are on the safe side. 

https://opensource.org/licenses/0BSD

Necessary to handle tslib 
https://github.com/fastify/releasify/pull/235#issuecomment-1202473519

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/plugins-ci-mysql.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
         run: npm i --ignore-scripts
 
       - name: Check Licenses
-        run: npx license-checker --production --onlyAllow="BSD-2-Clause;BSD-3-Clause;ISC;MIT;Apache-2.0"
+        run: npx license-checker --production --onlyAllow="BSD-2-Clause;BSD-3-Clause;ISC;MIT;Apache-2.0;0BSD"
```
Comment (Fdawgs) [inline .github/workflows/plugins-ci-mysql.yml:61]:
```suggestion
        run: npx license-checker --production --onlyAllow="0BSD;Apache-2.0;BSD-2-Clause;BSD-3-Clause;ISC;MIT"
```

May as well sort the licenses alphabetically 😄

File: .github/workflows/plugins-ci-postgres.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
         run: npm i --ignore-scripts
 
       - name: Check Licenses
-        run: npx license-checker --production --onlyAllow="BSD-2-Clause;BSD-3-Clause;ISC;MIT;Apache-2.0"
+        run: npx license-checker --production --onlyAllow="BSD-2-Clause;BSD-3-Clause;ISC;MIT;Apache-2.0;0BSD"
```
Comment (Fdawgs) [inline .github/workflows/plugins-ci-postgres.yml:61]:
```suggestion
        run: npx license-checker --production --onlyAllow="0BSD;Apache-2.0;BSD-2-Clause;BSD-3-Clause;ISC;MIT"
```

File: .github/workflows/plugins-ci.yml [hunk 3]
```diff
@@ -58,7 +58,7 @@ jobs:
         run: npm i --ignore-scripts
 
       - name: Check Licenses
-        run: npx license-checker --production --onlyAllow="BSD-2-Clause;BSD-3-Clause;ISC;MIT;Apache-2.0"
+        run: npx license-checker --production --onlyAllow="BSD-2-Clause;BSD-3-Clause;ISC;MIT;Apache-2.0;0BSD"
```
Comment (Fdawgs) [inline .github/workflows/plugins-ci.yml:61]:
```suggestion
        run: npx license-checker --production --onlyAllow="0BSD;Apache-2.0;BSD-2-Clause;BSD-3-Clause;ISC;MIT"
```

File: .github/workflows/plugins-ci-mysql.yml (hunk 1 above)
Comment (Uzlopak) [reply .github/workflows/plugins-ci-mysql.yml]: should not be a blocking issue ;)

**Pull request merged**